src/envoy/http/authn/authn_utils.cc

```diff
@@ -27,6 +27,10 @@ namespace AuthN {
 namespace {
 // The JWT audience key name
 static const std::string kJwtAudienceKey = "aud";
+// The JWT issuer key name
+static const std::string kJwtIssuerKey = "iss";
+// The key name for the APToken original claims
+static const std::string kAPTokenOriginalPayload = "original_claims";
```

Comment:
Shall we change the name to avoid "APToken", or explain in the comment that this is exchanged token?

Reply:
Done.